celery/backends/redis.py

```diff
@@ -408,6 +417,9 @@ def ConnectionPool(self):
     def client(self):
         return self._create_client(**self.connparams)
```

Comment:
I guess we could now call the new public method here (`create_client`), which could also be considered as a means of testing the new method, without actually adding any new test cases.